skore/src/skore/ui/project_routes.py

```diff
@@ -136,7 +137,7 @@ def __cross_validation_item_as_serializable(item: CrossValidationItem) -> dict:
         "plots": [
             {
                 "name": "cross-validation results",
-                "value": item.plot,
+                "value": plotly.io.to_json(item.plot),
```

Comment:
This will return a json string. Hence needs a double josn decoding. I do not think that it's what we want.

Reply:
gr